### ISSUE TITLE
RCL CSS path is still wrong (due to "_content" folder)

### ISSUE THREAD

**Issue by datvm** [mingyaulee/Blazor.BrowserExtension#77]
When an extension uses an RCL that includes a compiled CSS (like my [BlazorMaterialWeb](https://github.com/datvm/BlazorMaterialWeb/)), its `AppName.styles.css` would include an `@import`:

```css
@import '_content/BlazorMaterialWeb/BlazorMaterialWeb.bundle.scp.css';
```

![image](https://github.com/mingyaulee/Blazor.BrowserExtension/assets/6388546/8b6085ea-d240-49cf-beb8-b8a2fdbc83b1)

However, the correct path should be `content/...`.

A simple workaround is to include a correct path in your own CSS but it would be nice if it's fixed too.

```css
@import '/content/BlazorMaterialWeb/BlazorMaterialWeb.bundle.scp.css';
```

**Comment by mingyaulee** [comment]
That is certainly something that the library should include as well. I am currently working on the support of .Net 8. So after that, the next thing to do is including replacement of paths in the project's scoped CSS bundle imports.

**Comment by mingyaulee** [comment]
This is fixed in v0.12.2
